stable/redis-ha/templates/redis-haproxy-service.yaml

```diff
@@ -0,0 +1,31 @@
+{{- if .Values.haproxy.enabled }}
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ template "redis-ha.fullname" . }}-haproxy
+  labels:
+{{ include "labels.standard" . | indent 4 }}
+  annotations:
+  {{- if .Values.haproxy.service.annotations }}
+{{ toYaml .Values.haproxy.service.annotations | indent 4 }}
+  {{- end }}
+spec:
+  type: {{ default "ClusterIP" .Values.haproxy.service.type }}
+  {{- if or (not .Values.haproxy.service.type) (eq .Values.haproxy.service.type "ClusterIP") }}
+  clusterIP: None
```

Comment:
This doesn't need to be a headless service by default. Allowing it to have a clusterIP will enable load balancing across multiple HAproxy pods if they ever get scaled out. Really the headless service is mostly only useful for stateful sets where each pod needs to be accessed individually.

Reply:
I'll just remove this bit altogether